torch/_jit_internal.py

```diff
@@ -670,7 +669,7 @@ def __getitem__(self, types):
 # Retrieves a fully-qualified name (module hierarchy + classname) for a given obj.
 def _qualified_name(obj):
     # short-circuit in cases where the object already has a known qualified name
-    if isinstance(obj, torch.jit.ScriptFunction):
+    if hasattr(obj, "qualified_name"):
```

Comment:
This doesn't look like a safe change since `obj` is user provided and could have a `qualified_name` already